typescript/solver/config.ts

```diff
@@ -0,0 +1,15 @@
+import dotenvFlow from "dotenv-flow";
+
+dotenvFlow.config();
+
+const MNEMONIC = process.env.MNEMONIC;
```

Comment:
I wonder if there is an existing library which allows us to delegate key management and just gives us signers. Here is how we do it on the Hyperlane CLI https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/0a22e96553b7dbdaf8454cfd5f9e6290abd85d78/typescript/cli/src/context/context.ts#L59, but certianly would be cool if remote signers like AWS KMS could be used (nice to have for now)

Reply:
issue: https://github.com/BootNodeDev/intents-framework/issues/20